Google.PowerShell/ReleaseFiles/GoogleCloudPlatform.Format.ps1xml

```diff
@@ -162,5 +162,52 @@
         </TableRowEntries>
       </TableControl>
     </View>
+    <View>
+      <Name>BigQuery Data Row</Name>
```

Comment:
I think a better way would be for the output to look like a table like:
Book Name            Author          Published
-----------------------------------------------
Omelette              Shakespeare   1603

Do you think it would be possible?